src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/SelectOption.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.DraftApprenticeship.SelectOptionViewModel
+
+@{
+    ViewData["Title"] = "Select option";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <partial name="_ValidationSummary" />
+
+        <h1 class="govuk-heading-xl">Select a standard option</h1>
+
+        @*To do - Add link to standards cache*@
+        @*<div class="govuk-inset-text">
+            <p class="govuk-body">For more information visit ... </p>
+        </div>*@
+
+        <dl class="govuk-summary-list">
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key"> Apprenticeship training course </dt>
+                <dd class="govuk-summary-list__value"> @Model.StandardTitle </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key"> Version </dt>
+                <dd class="govuk-summary-list__value"> @Model.Version </dd>
+            </div>
+        </dl>
+
+        <form id="selectOption" method="post">
+            <div class="govuk-form-group @Html.AddClassIfPropertyInError(x => x.CourseOption, "govuk-form-group--error")">
+                @Html.AntiForgeryToken()
+                @Html.HiddenFor(x => x.DraftApprenticeshipId)
+                @Html.HiddenFor(x => x.FirstName)
+                @Html.HiddenFor(x => x.LastName)
+                @Html.HiddenFor(x => x.Email)
+                @Html.HiddenFor(x => x.Uln)
+                @Html.HiddenFor(x => x.CohortId)
+                @Html.HiddenFor(x => x.CohortReference)
```

Comment:
instead of `@Html.HiddenFor` can we use `<input type="hidden" asp-for="CohortReference"/>`